corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-history-custom.js

```diff
@@ -35,6 +35,12 @@ $(function () {
             // where State.data.tab won't be available
             link = $('a[data-toggle="tab"][href^="' + window.location.pathname + '"]');
             if (link.length !== 0) {
+                if (link.length > 1) {
```

Comment:
@amsagoff I'm not sure that deploy is worse than settings even for early app builders...I'd think they really want to be in a module or form page, but we don't know which one, so we can't really optimize (unless we did something like going back to the last page they visited, which would get more complex). App summary would be a great landing page except that because it has an unique nav it's hard to know where to go from there. Might be interesting to do something like default to settings until you make your first build, and then default to deploy.

Anyway, sounds like there isn't something obvious to do here & I'm just thinking out loud. Happy to leave it alone for now, appreciate your thoughts.